src/main/java/org/broadinstitute/dsm/route/AbstractionFormControlRoute.java

```diff
@@ -31,8 +31,9 @@ public Object processRequest(Request request, Response response, String userId)
         }
 
         if (StringUtils.isNotBlank(realm)) {
+            UserUtil userUtil = new UserUtil();
             if (RoutePath.RequestMethod.GET.toString().equals(request.requestMethod())) {
-                if (UserUtil.checkUserAccess(realm, userId, "mr_view")) {
+                if (userUtil.checkUserAccess(realm, userId, "mr_view", null)) {
```

Comment:
if you are already touch all classes to change this, can you also change the role string and get it from a nice central point ("mr_view", "mr_request" and all of them) to minimize typos in the check. thank you